### PR TITLE
Adjust section margins for the block edit page

### PR DESCRIPTION
**Pull request by ssandbekkhaug**
### Description

Adjust section margins for the block edit page.  [View spacing mocks](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=4093-7540&m=dev)

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Removed the release notes section if the title is sufficient for the release notes description, or put more details in that section.
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [x] Extended the README / documentation, if necessary. For user-facing features, consider updating [the user docs](https://github.com/civiform/docs). For "under-the-hood" changes or things more relevant to developers, consider updating [the dev wiki](https://github.com/civiform/civiform/wiki).

#### User visible changes

- [x] Followed steps to [internationalize](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#application-strings) any new strings
  - [x] Added context strings to new [messages](https://github.com/civiform/civiform/blob/main/server/conf/i18n/messages)
  - [x] Didn't use a message in applicant facing code that isn't translated yet (unless behind a flag)
- [x] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Made equivalent changes in Thymeleaf for applicant-facing features and changes (or created an issue in the [Northstar epic](https://github.com/orgs/civiform/projects/1/views/94) to track that it's missing from Thymeleaf)
- [x] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [x] Manually tested at 200% size
- [x] Manually evaluated tab order

### Issue(s) this completes

Part of #8626

### CODE REVIEW

**Comment by ssandbekkhaug** [comment]
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#8628](https://github.com/civiform/civiform/pull/8628) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8628?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>) 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: browser-test/image_snapshots/admin_question_preview_test/north-star-question-preview-page.png
Comment (melanie-exygy) [inline browser-test/image_snapshots/admin_question_preview_test/north-star-question-preview-page.png:1]: **suggestion (non-blocking):** The question looks squished up against the top to me in this admin view.  Would you mind adding a little padding, probably in the question container here?

File: server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss [hunk 1]
```diff
@@ -105,6 +105,16 @@ it is around the label */
   @include u-height('105');
 }
 
+// Between sections
+.section-external {
+  @include u-margin-top(4);
+}
+
+// Between elements within a section
+.section-internal {
```
Comment (azhangggggg) [inline server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:114]: question(non-blocker): is u-margin-top(3) going to be our default section margin within a section?

File: server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss [hunk 2]
```diff
@@ -105,6 +105,16 @@ it is around the label */
   @include u-height('105');
 }
 
+// Between sections
```
Comment (melanie-exygy) [inline server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:108]: **suggestion (non-blocking):** Is this just for sections in the block page or for sections everywhere in the app?  If it's just for the block view, would you mind adding that to the comment?

Comment (ssandbekkhaug) [reply server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:108]: It'll be used throughout the app

File: server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss (hunk 1 above)
Comment (ssandbekkhaug) [reply server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss:114]: Yes, see [the mocks](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=4093-7540&m=dev)

**Review by azhangggggg — APPROVED** [review]

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ssandbekkhaug** [comment]
* **#8637** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8637?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3  dependent PRs ([#8629](https://github.com/civiform/civiform/pull/8629) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8638](https://github.com/civiform/civiform/pull/8638) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8638?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#8639](https://github.com/civiform/civiform/pull/8639) <a href="https://app.graphite.dev/github/pr/civiform/civiform/8639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#8627** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8627?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ssandbekkhaug and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Pull request merged**